src/main/fc/stats.c

```diff
@@ -16,21 +18,24 @@
 #include "config/parameter_group_ids.h"
 
 #define MIN_FLIGHT_TIME_TO_RECORD_STATS_S 10    //prevent recording stats for that short "flights" [s]
+#define MIN_FLIGHT_DISTANCE_M 10    // minimum distance flown for a flight to be registered [m]
```

Comment:
How about increasing the distance to 30?

```suggestion
#define MIN_FLIGHT_DISTANCE_M 30    // minimum distance flown for a flight to be registered [m]
```

Reply:
I can change it to that, no problem; I can't imagine a flight going for less than 30m anyway